src/app/component/ExpandableContentShowMore/ExpandableContentShowMore.component.js

```diff
@@ -29,18 +29,47 @@ export class ExpandableContentShowMore extends PureComponent {
     };
 
     state = {
-        isOpen: false
+        isOpen: true
     };
 
+    isExpanding = false;
+
+    expandableContentHeight = 'auto';
+
+    expandableRef = createRef();
+
     ref = createRef();
 
-    handleShowAllButtonClick = () => {
-        const { pageYOffset } = window;
+    componentDidMount() {
+        this.expandableContentHeight = this.expandableRef.current.getBoundingClientRect().height;
+        this.setState({ isOpen: false });
```

Comment:
Interesting approach :D I would go for max-height manipulation, but this seem ok as well. Interestingly it does not lag upon reload.

Reply:
This is done to calculate hidden filter block height on first render to use it later as max-height.
I used a constant first, but then the max-height is not same to same as height and the animation doesn't look so good.